src/components/internals/localized-input-toggle/localized-input-toggle.js

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
```

Comment:
This is the component that replaces the ones duplicated for the 4 localized inputs affected in this PR.